### PR TITLE
clean the repo

### PR DESCRIPTION
**Pull request by mikhail-vlasenko**
remove 2 directories that we are not using, move job files, move visualization notebooks and files, update LICENSE

### CODE REVIEW

File: test_robustness.job
Comment (iasonsky) [inline test_robustness.job:1]: This is used to reproduce the barplots from Figure 4

File: test_robustness2.job
Comment (iasonsky) [inline test_robustness2.job:1]: This is used to reproduce the barplots from Figure 4 and appendix

**Review by iasonsky — APPROVED** [review]
lgtm, except the two jobfiles. Maybe I can combine them into one.

**Comment by mikhail-vlasenko** [comment]
feel free to commit the combination here

**Comment by mikhail-vlasenko** [comment]
@iasonsky feel free to commit the combination here

**Pull request merged**